src/ui/public/management/index.js

```diff
@@ -0,0 +1,23 @@
+import ManagementSection from './section';
+
+const sections = new ManagementSection('management', {
+  display: 'Management'
+});
+
+// TODO: where should this live?
+sections.register('data', {
+  display: 'Connect Data',
+  order: 0
+});
+
+sections.register('elasticsearch', {
+  display: 'Elasticsearch',
+  order: 10
+});
+
+sections.register('kibana', {
+  display: 'Kibana',
+  order: 20,
+});
```

Comment:
Personally, I'm 100% alright with this.  Trying to make a completely generic interface that can scale across any number of categories with any number of links is downright impossible to do well.  I think it makes sense for the management app to specifically control which categories can exist.